### PR TITLE
test: add e2e test for config labels and config envs

### PR DESCRIPTION
**Pull request by jrangelramos**
# Changes

- :broom: Adding e2e tests for config labels command
- :broom: Adding e2e tests for config envs command

/kind cleanup

### CODE REVIEW

File: test/_e2e/shell_cmd_runner.go [hunk 1]
```diff
@@ -95,6 +110,9 @@ func (f *TestShellCmdRunner) Exec(args ...string) TestShellCmdResult {
 	if f.SourceDir != "" {
 		cmd.Dir = f.SourceDir
 	}
+	if f.Env != nil {
+		cmd.Env = append(os.Environ(), f.Env...)
+	}
```
[the remark this answers is not in the document]
Comment (matejvasek) [reply test/_e2e/shell_cmd_runner.go:110]: Shouldn't we always set `os.Environ()` to forked process?

File: test/_e2e/cmd_config_envs_test.go [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"knative.dev/kn-plugin-func/k8s"
+)
+
+// setupConfigEnvsTest add to cluster config maps and secrets used by the test
+func setupConfigEnvsTest(t *testing.T) {
+
+	config, _ := k8s.GetClientConfig().ClientConfig()
```
Comment (matejvasek) [inline test/_e2e/cmd_config_envs_test.go:23]: Please do handle the errors.

File: test/_e2e/shell_cmd_runner.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (jrangelramos) [reply test/_e2e/shell_cmd_runner.go]: yes.. that won't hurt and will look cleaner, thanks. I see go does that when we leave it empty.. but that's ok!

File: test/_e2e/cmd_config_envs_test.go [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+
+	// Add Secret
+	secret := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{Name: "test-secret"},
+		Data: map[string][]byte{
+			"TEST_SECRET_PW1": []byte("pw1"),
+			"TEST_SECRET_PW2": []byte("pw2"),
+		},
+	}
+	_, err = clientset.CoreV1().Secrets(namespace).Create(context.TODO(), secret, metav1.CreateOptions{})
```
[the remark this answers is not in the document]
Comment (jrangelramos) [reply test/_e2e/cmd_config_envs_test.go]: sure! thanks

**Pull request merged**